Godeps/_workspace/src/github.com/Clever/gearadmin/Makefile

```diff
@@ -0,0 +1,20 @@
+SHELL := /bin/bash
+PKG = github.com/Clever/gearadmin
+SUBPKGS =
```

Comment:
In the spirit if "Don't Trust The Textbook", I'm going to ask "why?" The way I see it, the biggest benefit of godeps is locking down to a specific commit, which is unrelated to whether or not you check that code into your repo.

The upside I see to checking it in: faster build times + test times, since you don't need to redownload (you could also get some of this benefit in other ways, e.g. by having Drone cache the `_workspace` directory)
The downside: bloated repositories + commits